include/ClipView.h

```diff
@@ -244,8 +244,13 @@ protected slots:
 	TimePos draggedClipPos( QMouseEvent * me );
 	int knifeMarkerPos( QMouseEvent * me );
 	void setColor(const std::optional<QColor>& color);
-	//! Return true iff the clip could be split. Currently only implemented for samples
-	virtual bool splitClip( const TimePos pos ){ return false; };
+	
+	/**
+	* Split this Clip into two clips
+	* @param pos the position of the split, relative to the start of the clip
+	* @return true iff the clip could be split
```

Comment:
```suggestion
	* @return true if the clip could be split
```

Reply:
Do you think this was a typo or did the original author intend "iff" to mean "if and only if"?